### PR TITLE
Fix thank you message on submit

### PR DESCRIPTION
**Pull request by ujjwal5**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--ujjgupta-test-repo--ujjwal5.hlx.page/content/forms/af/ujjgupta/wizard-single

- After: https://fix-reset--ujjgupta-test-repo--ujjwal5.hlx.page/content/forms/af/ujjgupta/wizard-single

### CODE REVIEW

File: blocks/form/form.js [hunk 1]
```diff
@@ -414,8 +414,15 @@ export async function createForm(formDef, data) {
   }
 
   form.addEventListener('reset', async () => {
+    const thankYouMessage = form.querySelector('.form-message.success-message');
```
Comment (vdua) [inline blocks/form/form.js:417]: I was thinking of updating the following html structure
```
<form>
<div success-message>
</div>
<div form-content>
</div>
</form>
```
WDYT @jalagari ?

Comment (jalagari) [reply blocks/form/form.js:417]: Even I was thinking of html structure but we were facing some styline issue but we revisit it again

**Comment by nit23uec** [comment]
@ujjwal5, can you pls add a test case also for this?

**Review by nit23uec — CHANGES_REQUESTED** [review]
test case missing.

**Comment by ujjwal5** [comment]
Closing this PR in favour of https://github.com/adobe-rnd/aem-boilerplate-forms/pull/91

**Pull request closed without merging**